oak_functions/loader/src/tests.rs

```diff
@@ -82,13 +81,18 @@ async fn test_long_response_time() {
     // So we expect the request to fail, with `response not available error`.
     let scenario = |server_port: u16| async move {
         let result = make_request(server_port, br#"{"lat":52,"lon":0}"#).await;
-        // TODO(#1987): Check elapsed time
+        // Check the elapsed time, allowing a margin of 10ms.
+        assert!(
+            result.elapsed < Duration::from_millis(allowed_processing_time + 10),
```

Comment:
This should be adding Durations, not numbers. Something like

```rust
let margin = Duration::from_millis(10);
let constant_processing_time = Duration::from_millis(10);
...
assert!(result.elapsed < constant_processing_time + margin);
```

Reply:
Thanks. Done.